consensus/istanbul/validator/weighted.go

```diff
@@ -95,6 +95,7 @@ func newWeightedValidator(addr common.Address, reward common.Address, votingpowe
 
 type weightedCouncil struct {
 	subSize    uint64
+	demoted    istanbul.Validators
```

Comment:
```suggestion
	demotedValidator    istanbul.Validators
```

Reply:
How about add `stakedValidator` instead of `demotedValidator`?
So `stakedValidator` like filtered validators and origin validator list can be managed.

Reply:
So, the `validators` and `demoted` are exclusive. The `validators` are the filtered validators.